nbconvert/exporters/templateexporter.py

```diff
@@ -283,6 +284,8 @@ def _create_environment(self):
             extensions=JINJA_EXTENSIONS
             )
 
+        environment.globals['uuid4'] = uuid.uuid4
```

Comment:
Shouldn't this be `lambda : uuid.uuid4().hex` returning a string instead of the instance?